### PR TITLE
add easyconfig for foss/2015a + HPL for testing

### PR DESCRIPTION
**Pull request by boegel**
Proposal for the `foss/2015a` "common toolchain", consisting of:

* GCC version 4.9.2 (latest to date, Oct 23rd 2014)
* OpenMPI version 1.8.3 (latest to date, Sept 26th 2014)
* OpenBLAS 0.2.13 (latest to date, Dec 3rd 2014)
* FFTW 3.3.4 (latest to date, March 16 2014)
* ScaLAPACK 2.0.2 (latest to date, May 1st 2012)

Latest available versions of all components were selected keeping in mind that they are required for several important features, including (full) C++11 support, OpenMP v4.
This tips the scale in favour of most recent releases rather than being conservative and going for the last update of the previous major release.

Results from testing this toolchain by rebuilding existing `goolf`/`foss` easyconfigs with `foss/2015a` will be posted soon.

### CODE REVIEW

**Comment by hajgato** [comment]
Bump to OpenMPI 1.8.4?

File: easybuild/easyconfigs/h/hwloc/hwloc-1.10.0-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name = 'hwloc'
+version = '1.10.0'
```
Comment (boegel) [inline easybuild/easyconfigs/h/hwloc/hwloc-1.10.0-GCC-4.9.2.eb]: @bgoglin: can you shed some light on when hwloc v1.10.1 would be released?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 9 out of 10 (10 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1d68aa08fca100367dca for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/34f767310ead77c3b6e8 for a full test report.

File: easybuild/easyconfigs/h/hwloc/hwloc-1.10.0-GCC-4.9.2.eb (hunk 1 above)
Comment (bgoglin) [inline easybuild/easyconfigs/h/hwloc/hwloc-1.10.0-GCC-4.9.2.eb]: Sorry guys, v1.10.1 was ready but we got a late AIX bug that I couldn't fix before the hollidays. If I get the required AIX access today, I can do v1.10.1 by the end of the week. Hopefully the admins of these AIX platform will process my account request soon.

Comment (boegel) [inline easybuild/easyconfigs/h/hwloc/hwloc-1.10.0-GCC-4.9.2.eb]: Thanks @bgoglin for the info!

**Comment by boegel** [comment]
still no hwloc v1.10.1; it seems like it doesn't feature any major bug fixes for 'normal' x86-64-based Linux clusters (see discussion with hwloc release manager at https://twitter.com/kehoste/status/552500451393159168), so merging this in as is with hwloc v1.10.0 (after agreeing with the VSC parties involved)

**Pull request merged**